wiz-broker/values.yaml

```diff
@@ -8,11 +8,15 @@ namespace: "kube-system"
 installRbac: false
 installBroker: false
 
+rbacServiceAccount:
+  name: "wiz-kube-connector"
+  annotations: {}
+
 serviceAccount:
+  create: false
   # Annotations to add to the service account
   annotations: {}
   # The name of the service account to use.
-  name: "wiz-kube-connector"
```

Comment:
Add default name for the service account 
https://github.com/wiz-sec/charts/blob/1267e4c5e55e52e5a9e27111a26c26790f495442/wiz-broker/templates/_helpers.tpl#L39